.travis.yml

```diff
@@ -1,5 +1,10 @@
+os:
+  - linux
 language: node_js
 node_js:
-  - "0.10"
+  - "10.16.0"
```

Comment:
Given current Meteor version this should be the latest Node 8.

Reply:
For future compatibility (with Meteor 1.9) we might also include node v12 for testing...

Reply:
I will try changing this to 8.15.1 and we should create a ticket for including any future requirements.

Reply:
If you want you can add latest Node 12 to the list to account for Meteor 1.9.